a1/README.md

```diff
@@ -425,12 +427,30 @@ While this guide explains the *what*, *why* and *how*, I find it helpful to see
 
   ```html
   <!-- avoid -->
-  <input ng-model="customerProductItemVm.text">
+  <div ng-controller="SomeController as customerProductItemVm">
+    <input ng-model="customerProductItemVm.title">
+  </div>
   ```
 
   ```html
   <!-- recommended -->
-  <input ng-model="productVm.id">
+  <div ng-controller="SomeController as productVm">
+    <input ng-model="productVm.title">
+  </div>
+  ```
+
+  Note: In this case, you'll need to use `productVm.*` to watch controller's member properly:
```

Comment:
I'm pretty sure I've mentioned it before, but imo hatdcoding the alias used in a template inside the JS implementation is a bad idea.